### ISSUE TITLE
[priority:HIGH] convert existing website to react website

### ISSUE THREAD

**Issue by devvsakib** [devvsakib/power-the-web#66]
Hi, right now we have a static and normal HTML website. But, it's time to go higher. We are thinking to develop this open-source website into a dynamic react js website. 
For this, I will be helpful if you collaborate with me. We will convert our work between us. For example, if I want to create a navbar, I will work on the header components, and you will work on the desired component you want.

So basically, anyone can be assigned to this issue. There is no limit.

### Requirement
Have to good understanding of JavaScript, ReactJs (no need  professional, just know how react works and some knowledge about react components), Tailwind CSS, react styled-components.

That's it...!!!

You can join discord - [JOIN](https://discord.gg/xwRbDwnbMr)

**Comment by yes-im-harsh** [comment]
I want to work on this, Please Assign it to me @devvsakib

**Comment by devvsakib** [comment]
> I want to work on this, Please Assign it to me @devvsakib



> I want to work on this, Please Assign it to me @devvsakib

sure, navigate there #75

**Comment by udaykahar** [comment]
Can i also want to contribute in this..
